generic/tclExtend.h

```diff
@@ -52,6 +52,10 @@ typedef void *void_pt;
 #define TCLX_CMDL_INTERACTIVE	(1<<0)
 #define TCLX_CMDL_EXIT_ON_EOF	(1<<1)
 
+#define CONST const
+#define VOID void
+#define panic Tcl_Panic
+
```

Comment:
It needs to build without these lines. If there are other changes necessary to make it work they need to be made elsewhere.